test/unit/basic.js

```diff
@@ -0,0 +1,268 @@
+module( "basic", {
+	teardown: moduleTeardown
+} );
+
+if ( jQuery.ajax ) {
+test( "ajax", function( assert ) {
+	expect( 4 );
+
+	var done = assert.async(),
+		requestsLeft = 3;
+
+	jQuery.ajax({
+		type: "GET",
+		url: url( "data/name.php?name=foo" ),
+		success: function( msg ) {
+			assert.strictEqual( msg, "bar", "Check for GET" );
+			requestsLeft--;
+			if ( !requestsLeft ) {
+				done();
```

Comment:
Updated, although that doesn't save that much. Ideally `assert.async()` would accept a parameter indicating how many async operations are expected and that would require calling `done()` this many times.

BTW, `done.pop().call()` is not needed, it's just `done.pop()()` after all.